### DISCUSSION TITLE
:hugs: Document how to upload a dataset to HuggingFace

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#10368] [Ideas]
# :grey_question: About

[HF indicates that `duckdb` makes it possible to push data to a HF dataset:
](https://huggingface.co/docs/hub/datasets-libraries)

![image](https://github.com/duckdb/duckdb/assets/5235127/47e74b7d-3726-46bc-ae56-19e69b878f49)

# :dart: Actions

Share a code snippet that show how to achieve this.

**Comment by Mause** [comment]
It might be possible if they implement S3 or fsspec, but that's really more a question for them than for us

**Reply by adriens** [reply]
:ok: that matches much more my understanding :pray:

**Reply by severo** [reply]
This documentation page explains how to upload a dataset to HF using duckdb: https://huggingface.co/docs/hub/datasets-duckdb. 

Your question was more about having the corresponding code snippet in the duckdb docs? (would be awesome, happy to give a hand)

**Reply by adriens** [reply]
> Your question was more about having the corresponding code snippet in the duckdb docs? (would be awesome, happy to give a hand)

Indeed yes :smile_cat:
